cli.js

```diff
@@ -42,7 +48,34 @@ function init(data) {
 }
 
 if (process.stdin.isTTY || cli.flags.stdin === false) {
-	if (cli.flags.template) {
+	if (cli.flags.demo) {
+		const demoStr = [
+			chalk.bold('bold'),
+			chalk.dim('dim'),
+			chalk.italic('italic'),
+			chalk.underline('underline'),
+			chalk.inverse('inverse'),
+			chalk.strikethrough('strikethrough'),
+			chalk.black('black'),
+			chalk.red('red'),
+			chalk.green('green'),
+			chalk.yellow('yellow'),
+			chalk.blue('blue'),
+			chalk.magenta('magenta'),
+			chalk.cyan('cyan'),
+			chalk.white('white'),
+			chalk.gray('gray'),
+			chalk.bgBlack('bgBlack'),
+			chalk.bgRed('bgRed'),
+			chalk.bgGreen('bgGreen'),
+			chalk.bgYellow('bgYellow'),
+			chalk.bgBlue('bgBlue'),
+			chalk.bgMagenta('bgMagenta'),
+			chalk.bgCyan('bgCyan'),
+			chalk.bgWhite('bgWhite')
```

Comment:
This can be simplified by putting the names in an array, looping over it and calling the chalk method by that name.